NSIS/install.nsi

```diff
@@ -120,37 +122,39 @@
 
 ;--------------------------------
 
+!ifndef BASE_DIR
+	!define BASE_DIR "..\out"
+!endif
+
+!ifndef FILENAME
+	!define FILENAME "com0com_v4.0.0_w10_x64_signed.exe"
```

Comment:
should use a cmake configure file for this?
can be set up through cmake nsis package?